lib/pharos/phases/validate_cluster_hosts.rb

```diff
@@ -2,10 +2,14 @@
 
 module Pharos
   module Phases
-    class ValidateHostname < Pharos::Phase
-      title "Validate unique hostnames for all hosts"
+    class ValidateClusterHosts < Pharos::Phase
```

Comment:
All cluster wide host validations should be under this phase. New title should make it more clear.